test/unit/controller/user-controller.ts

```diff
@@ -906,6 +906,20 @@ describe('UserController', (): void => {
       expect(user.firstName).to.deep.equal(firstName);
       verifyUserResponse(spec, user);
     });
+    it('should allow user to update extensiveDataProcessing', async () => {
```

Comment:
Maybe also add a sanity-check use case that checks that a) you cannot change any other properties of yourself when you are not an admin and b) you cannot change extensive data processing of other users? Because I'm a bit scared

Reply:
Doesn't really work in the test-env as we define our own rulesets for each token per test file. This means that you will only be testing if I wrote the correct test cases, but it won't actually test if the correct RBAC is defined in the production database.

I'll make an issue and approve this.